### PR TITLE
Post refresh sweep

### PR DESCRIPTION
**Pull request by cdolm92**
# What

- Fixed truncated German text for smaller devices.
- Added new reward title text informing user of project deadline.
- removed "Back this project below" and "X rewards"

# Why

Part of the refresh.

# See 👀

See screenshots below.

### CODE REVIEW

**Comment by justinswart** [comment]
@cdolm92 hey hey what was the latest on this PR, should we merge `master` to resolve conflicts?

**Comment by cdolm92** [comment]
@justinswart we were waiting on shortened German string translations which we just got this morning!

File: Kickstarter-iOS/Locales/es.lproj/Localizable.strings [hunk 1]
```diff
@@ -292,7 +292,11 @@
 "The_replay_will_start_soon" = "Repetición comienza en breve";
 "There_was_a_problem_with_this_payment" = "Ocurrió un error con el pago.";
 "This_facebook_account_is_already_linked_to_another_Kickstarter_user" = "Esta cuenta de Facebook ya está asociada a otro usuario de Kickstarter.";
+<<<<<<< HEAD
 "This_project_will_only_be_funded_on_if_at_least_amount_is_pledged_by_date" = "Este proyecto sólo será financiado si se contribuye al menos un monto de %{amount} antes del %{date}.";
+=======
```
Comment (Scollaco) [inline Kickstarter-iOS/Locales/es.lproj/Localizable.strings]: Could you remove this conflict mark?

File: Library/Strings.swift [hunk 2]
```diff
@@ -4616,10 +4616,17 @@ catch your eye?"
...
    - **es**: "Este proyecto sólo será financiado si se contribuye al menos un monto de %{amount} antes del %{date}."
    - **ja**: "このプロジェクトは%{date} までに、少なくとも%{amount}プレッジされるでしょう。"
    - **de**: "Dieses Projekt wird nur finanziert, wenn bis am %{date} Unterstützungsbeiträge von mindestens %{amount} geleistet werden."
    - **fr**: "Ce projet ne sera financé que si au moins %{amount} sont engagés d'ici le %{date}."
+=======
+   - **es**: "This project will only be funded on if at least %{amount} is pledged by %{date}."
+   - **ja**: "This project will only be funded on if at least %{amount} is pledged by %{date}."
+   - **de**: "This project will only be funded on if at least %{amount} is pledged by %{date}."
+   - **fr**: "This project will only be funded on if at least %{amount} is pledged by %{date}."
+>>>>>>> wip
```
Comment (Scollaco) [inline Library/Strings.swift]: And these lines

File: Kickstarter-iOS/Locales/es.lproj/Localizable.strings [hunk 3]
```diff
@@ -292,7 +292,11 @@
...
+"This_project_will_only_be_funded_on_if_at_least_amount_is_pledged_by_date" = "This project will only be funded on if at least %{amount} is pledged by %{date}.";
+>>>>>>> wip
```
Comment (Scollaco) [inline Kickstarter-iOS/Locales/es.lproj/Localizable.strings]: Could you delete these lines? It probably appeared when we were rebasing.

**Pull request merged**